README.md

```diff
@@ -6,11 +6,13 @@ On the Live Video platform, Twitch, we are a group of broadcasters who write cod
 
 We want to ensure that broadcasters on the team are makers who have made a commitment to broadcast on the Twitch platform with a positive and welcoming atmosphere.  Members and prospective members:
 
+  1. Are least 18 years of age
   1. Twitch Affiliate or Partner status
+  1. Minimum of 5 average viewers over the last 30 days
   1. During broadcasts, politely answer questions from viewers
-  1. Promote a positive, inclusive, and welcoming chat room
-  1. Are a person, not a brand or company account
+  1. Promote a positive, inclusive, and welcoming chat room (we discourage Blue Comedy/Ribaldry)
```

Comment:
can we make this ".. we discourage excessive Blue Comedy..."

Reply:
Doing it now!